src/app/(post-login)/submission/program/[shortName]/clinical-data/page.tsx

```diff
@@ -79,7 +83,7 @@ const ClinicalDataPageComp = ({
 		'tab',
 		defaultClinicalEntityTab,
 	);
-	const [selectedDonors, setSelectedDonors] = useUrlParamState('donorId', donorId);
+	const [selectedDonors, setSelectedDonors] = useUrlParamState('donorId', 'donorId');
```

Comment:
This change I assume is just a mock for testing? No way 'donorId' is the actual value we want